draft-ietf-mops-streaming-opcons.md

```diff
@@ -274,6 +305,23 @@ The combination of the large volume of "torrents" and the peer-to-peer character
 
 Especially as end users increase use of video-based social networking applications, it will be helpful for access network providers to watch for increasing numbers of end users uploading significant amounts of content. 
 
+##Extremely Unpredictable Usage Profiles
+
+The causes of unpredictable usage described in {{sec-unpredictable}} were more or less the result of human choices, but we were reminded during a post-IETF 107 meeting that humans are not always in control, and forces of nature can cause enormous fluctuations in traffic patterns.
+
+In his talk, Sanjay Mishra {{Mishra}} reported that after the CoViD-19 pandemic broke out in early 2020,
+ 
+- Comcast’s streaming and web video consumption rose by 38%, with their reported peak traffic up 32% overall between March 1 to March 30 {{Comcast}},
+- AT&T reported a 28% jump in core network traffic (single day in April, as compared to pre stay-at-home daily average traffic), with video accounting for nearly half of all mobile network traffic, while
+social networking and web browsing remaining the highest percentage (almost a quarter each) of overall mobility traffic {{ATT}}, and 
+- Verizon reported similar trends with video traffic up 36% over an average day (pre COVID-19) {{Verizon}}.
+
+We note that other operators saw similar spikes during this time period. Craig Labowitz {{Labovitz}} reported 
+
+- Weekday peak traffic increases over 45%-50% from pre-lockdown levels, 
+- A 30% increase in upstream traffic over their pre-pandemic levels, and
+- A steady increase in the overall volume of DDoS traffic, with amounts exceeding the pre-pandemic levels by 40%. (He attributed this increase to the significant rise in gaming-related DDoS attacks, as gaming usage also increased.)
```

Comment:
The parenthetical might need to either get left out or include a reference to explain "gaming-related DDoS attacks", which the primary reference itself doesn't do.  Maybe this?
https://venturebeat.com/2018/05/13/why-the-game-industry-is-still-vulnerable-to-distributed-denial-of-service-attacks/

Reply:
I agree, and I added the reference you suggested.